server.R

```diff
@@ -179,24 +179,33 @@ shinyServer(function(input, output, session) {
   
   
   # POLICY TAB
-  # plot the network for visual representation
-  output$policyTabNetwork <- renderPlot({
-    graphviz.plot(stable.fit,  
-                  layout = "dot",
-                  shape = "ellipse",
-                  render = TRUE)
-  })
+  # list the nodes checklist dynamically based on model instead of hardcoding
+  
+  # extract the nodes from the model
```

Comment:
I think there is a nodes function you can use to extract nodes from model without using a loop. 
nodes(model.fit)

Reply:
I ended up changing it, works the same. I also changed some variable names to make it easier to read.